### ISSUE TITLE
Can the Event Listener for 'online' which triggers a reload be configurable?

### ISSUE THREAD

**Issue by g-farrow** [shadowwalker/next-pwa#232]
# Summary

The behaviour of next-pwa which causes the page to be reloaded when the browser goes back "online" is not always the desired behaviour. When this happens it causes the app to lose its most recent state and interrupts the Users experience.

The code which causes this is here: https://github.com/shadowwalker/next-pwa/blob/e1d312927c43f45bbb9b07633513dd2c64d5ff79/register.js#L77-L79

Is there a reason why this code exists (i.e. is it solving another problem)? If it was to be removed/disable would it cause other unexpected behaviour or defects?

Could this behaviour be configurable?

# Versions

- `next-pwa`: 5.2.21
- `next`: 11.0.0

# How To Reproduce

1. Go to a next-pwa app
2. Switch the network to "offline" in Dev Tools
3. Switch the network to "online" in Dev Tools
4. See the page reload

# Expected Behaviors

Ideally, we can set a new flag in `next.config.js` e.g.: 

```js
module.exports = withPWA({
  pwa: {
    dest: 'public',
    swSrc: '/src/service-worker/index.js',
    cacheOnFrontEndNav: true,
    reloadOnOnline: false
  }
})
```

# Additional Context

If this sounds reasonable, I'm happy to open a PR 👍

**Comment by shadowwalker** [comment]
Go ahead for the PR

**Comment by g-farrow** [comment]
Thanks @shadowwalker - I've created PR #236

**Comment by devfaic** [comment]
@shadowwalker Waiting for the new version with this change included. Thanks @g-farrow for the solution.

**Comment by shadowwalker** [comment]
Published in `5.2.24`
